Plugins/Skinning/src/SkinningComponent.cpp

```diff
@@ -1,65 +1,67 @@
 #include <SkinningComponent.hpp>
 
-#include <Core/Animation/Pose/PoseOperation.hpp>
 #include <Core/Geometry/Normal/Normal.hpp>
+#include <Core/Animation/Pose/PoseOperation.hpp>
```

Comment:
please reorder includes (or use the scripts to have git re-format the code according to the defined style )

Reply:
The includes have been reordered as well.